src/kernels/composable_kernel/include/kernel_algorithm/gridwise_convolution_backward_data_implicit_gemm_v4r1_xdlops_gnchw_gkcyx_gnkhw.hpp

```diff
@@ -26,18 +27,17 @@ template <index_t GridSize,
           index_t GemmMPerBlock,
           index_t GemmNPerBlock,
           index_t GemmKPerBlock,
+          index_t GemmKPACK,
           index_t GemmMPerWave,
           index_t GemmNPerWave,
-          index_t GemmThreadGemmDataPerReadM,
-          index_t GemmThreadGemmDataPerReadN,
-          typename GemmABlockCopyThreadSliceLengths_GemmG_GemmK_GemmM,
-          typename GemmABlockCopyThreadClusterLengths_GemmG_GemmK_GemmM,
+          typename GemmABlockCopyThreadSliceLengths_GemmG_GemmK_GemmM_GemmKPACK,
+          typename GemmABlockCopyThreadClusterLengths_GemmG_GemmK_GemmM_GemmKPACK,
           index_t GemmABlockCopySrcDataPerRead_GemmM,
-          index_t GemmABlockCopyDstDataPerWrite_GemmM,
-          typename GemmBBlockCopyThreadSliceLengths_GemmG_GemmK_GemmN,
-          typename GemmBBlockCopyThreadClusterLengths_GemmG_GemmK_GemmN,
+          index_t GemmABlockCopyDstDataPerWrite_GemmKPACK,
+          typename GemmBBlockCopyThreadSliceLengths_GemmG_GemmK_GemmN_GemmKPACK,
+          typename GemmBBlockCopyThreadClusterLengths_GemmG_GemmK_GemmN_GemmKPACK,
```

Comment:
why are these changes required to remove the solvers ?

Reply:
I removed some file and change some file name. Git diff thought I modified files